frontend/packages/shared/defi/unit.ts

```diff
@@ -0,0 +1,14 @@
+import BigNumber from "bignumber.js";
+
+export function toChainIdUnit(value: number | BigNumber) {
```

Comment:
may be tell if Composable chain. It can be different from other chains

Reply:
Yup, will do that.